lib/stdlib/doc/src/peer.xml

```diff
@@ -0,0 +1,319 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!DOCTYPE erlref SYSTEM "erlref.dtd">
+
+<!-- %ExternalCopyright% -->
+
+<erlref>
+  <header>
+    <copyright>
+      <year>2021</year><year>2021</year>
+      <holder>Maxim Fedorov, WhatsApp Inc.</holder>
+    </copyright>
+    <legalnotice>
+      Licensed under the Apache License, Version 2.0 (the "License");
+      you may not use this file except in compliance with the License.
+      You may obtain a copy of the License at
+ 
+          http://www.apache.org/licenses/LICENSE-2.0
+
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    
+    </legalnotice>
+
+    <title>peer</title>
+    <prepared>maximfca@gmail.com</prepared>
+    <docno></docno>
+    <date></date>
+    <rev></rev>
+    <file>peer.xml</file>
+  </header>
+  <module since="OTP 25.0">peer</module>
+  <modulesummary>Start and control linked Erlang nodes.
+  </modulesummary>
+  <description>
+    <p>This module provides functions for starting linked Erlang nodes.
+      When origin node (the one started linked peers) terminates, all
+      linked peers are also terminated.
+      Depending on the start mode, terminal output and file I/O
+      of linked peers may go through the origin.</p>
+
+    <p>Peer nodes on other hosts than the current one are started with
```

Comment:
This seems to be contradicted later in that `rsh` is the default. In my opinion, support for `rsh` should be removed, but the OTB will have to decide.

Reply:
That'd be great if `rsh` support can be dropped. However I will still keep some way to amend/process path to executable and command line, because it's needed to support virtualised environments. I will provide examples in the documentation demonstrating intended usage, and add corresponding test cases. In short, instead of ssh host_b erl -name node2`, I use it to run `docker run -it otp -name node_in_docker`.